mindmeld/components/_config.py

```diff
@@ -1028,135 +1026,46 @@ def get_auto_annotator_config(app_path=None):
-        + "[^\w\d\s"  # noqa: W605
-        + exception_chars
-        + "]+(?=s)"
-    )
-
-    keep_special_regex_list.append("?P<underscore>_")  # noqa: W605
-    keep_special_regex_list.append("?P<begspace>^\s+")  # noqa: W605
-    keep_special_regex_list.append("?P<trailspace>\s+$")  # noqa: W605
-    keep_special_regex_list.append("?P<spaceplus>\s+")  # noqa: W605
-    keep_special_regex_list.append("?P<apos_space> '|' ")  # noqa: W605
-    keep_special_regex_list.append("?P<apos_s>(?<=[^\\s])'[sS]")  # noqa: W605
-    # handle the apostrophes used at the end of a possessive form, e.g. dennis'
-    keep_special_regex_list.append("?P<apos_poss>(^'(?=\S)|(?<=\S)'$)")  # noqa: W605
-
-    return keep_special_regex_list
-
-
-def get_tokenizer_config(app_path=None, exclude_from_norm=None):
-    """Gets the tokenizer configuration for the app at the specified path.
+def get_active_learning_config(app_path=None):
+    """Gets the active learning configuration for the app at the specified path.
 
     Args:
         app_path (str, optional): The location of the MindMeld app
-        exclude_from_norm (list, optional): chars to exclude from normalization
 
     Returns:
-        dict: The tokenizer configuration.
+        dict: The active learning configuration.
     """
-    DEFAULT_TOKENIZER_CONFIG["default_allowed_patterns"] = _get_default_regex(
-        exclude_from_norm
-    )
 
     if not app_path:
-        return DEFAULT_TOKENIZER_CONFIG
+        return DEFAULT_ACTIVE_LEARNING_CONFIG
     try:
-        tokenizer_config = getattr(
-            _get_config_module(app_path), "TOKENIZER_CONFIG", DEFAULT_TOKENIZER_CONFIG
+        active_learning_config = getattr(
+            _get_config_module(app_path),
+            "ACTIVE_LEARNING_CONFIG",
+            DEFAULT_ACTIVE_LEARNING_CONFIG,
         )
-        if not tokenizer_config.get("allowed_patterns"):
-            # If allowed_patterns are not provided, use default
-            tokenizer_config["allowed_patterns"] = []
-            tokenizer_config["default_allowed_patterns"] = _get_default_regex(
-                exclude_from_norm
-            )
-        return tokenizer_config
+        return active_learning_config
     except (OSError, IOError, AttributeError):
-        logger.info("No app configuration file found. Using default tokenizer config.")
-        return DEFAULT_TOKENIZER_CONFIG
+        logger.info("No app configuration file found.")
+        return DEFAULT_ACTIVE_LEARNING_CONFIG
 
 
-def get_active_learning_config(app_path=None):
-    """Gets the active learning configuration for the app at the specified path.
+def get_text_preparation_config(app_path=None):
+    """Gets the text preparation configuration for the app at the specified path.
 
     Args:
         app_path (str, optional): The location of the MindMeld app
 
     Returns:
-        dict: The active learning configuration.
+        dict: The tokenizer configuration.
```

Comment:
its unclear why the function states `get_text_preparation_config ` but we give back a tokenizer? I thought the tokenizer was part of the text prep?

Reply:
I think we are returning a text prep pipeline here, if thats the case, lets not mention tokenizer.

Reply:
Changed to `dict: The text preparation pipeline configuration.`